app/pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dependencies:
   skeletonizer: ^1.1.1
   flutter_chat_ui: ^1.6.9
   flutter_chat_types: ^3.3.2
+  scroll_to_index: ^3.0.1
```

Comment:
I believe ```flutter_chat_ui``` support scroll to unread [feature](https://pub.dev/documentation/flutter_chat_ui/latest/flutter_chat_ui/ScrollToUnreadOptions-class.html), if its used for that case specifically...

Reply:
It comes with the flutter_chat_ui but needed to add it so I can connect to the scroll controller as the chat ui doesn't expose the type...